Tests fail on Celery for Python 3.7 only

```pytb
=================================== FAILURES ===================================
__________________________________ test_known __________________________________

    def test_known():
        for known_backend in [
            "celery",
            "multiproc:4",
            "multiproc:auto",
            "foregroundasync",
            "externalasync:default",
        ]:
>           b = backend_from_string(known_backend)

tests/test_backends.py:13: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
packtivity/backendutils.py:192: in backend_from_string
    return backends[k]["default"](backendstring, backendopts)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

backendstring = 'celery', backendopts = {}

    @backend("celery")
    def celery_backend(backendstring, backendopts):
>       backend = asyncbackends.CeleryBackend(**backendopts)
E       AttributeError: module 'packtivity.asyncbackends' has no attribute 'CeleryBackend'

packtivity/backendutils.py:132: AttributeError
_________________________________ test_celery __________________________________

    def test_celery():
>       from packtivity.asyncbackends import CeleryProxy
E       ImportError: cannot import name 'CeleryProxy' from 'packtivity.asyncbackends' (/home/runner/work/packtivity/packtivity/packtivity/asyncbackends.py)

tests/test_proxies.py:5: ImportError
```

--- a/setup.py
+++ b/setup.py
@@ -37,9 +37,9 @@ setup(
     install_requires=deps,
     extras_require={
         "celery": [
-            "celery",
+            "celery>=4.0.0; python_version >= '3.8'",
+            "celery<=4.2.2; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
             "redis",
-            "importlib-metadata<5.0.0; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
         ]
     },
     entry_points={

--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ setup(
     install_requires=deps,
     extras_require={
         "celery": [
-            "celery==4.2.2",
+            "celery>=5.0.0",
             "redis",
             "importlib-metadata<5.0.0; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
         ]

--- a/setup.py
+++ b/setup.py
@@ -35,7 +35,13 @@ setup(
     python_requires=">=3.7",
     include_package_data=True,
     install_requires=deps,
-    extras_require={"celery": ["celery", "redis"]},
+    extras_require={
+        "celery": [
+            "celery",
+            "redis",
+            "importlib-metadata<5.0.0; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
+        ]
+    },
     entry_points={
         "console_scripts": [
             "packtivity-run=packtivity.cli:runcli",

--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ setup(
     install_requires=deps,
     extras_require={
         "celery": [
-            "celery",
+            "celery>=4.2.2",
             "redis",
             "importlib-metadata<5.0.0; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
         ]

--- a/setup.py
+++ b/setup.py
@@ -37,9 +37,9 @@ setup(
     install_requires=deps,
     extras_require={
         "celery": [
-            "celery>=4.0.0; python_version >= '3.8'",
-            "celery<=4.2.2; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
+            "celery",
             "redis",
+            "importlib-metadata<5.0.0; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
         ]
     },
     entry_points={

--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ setup(
     install_requires=deps,
     extras_require={
         "celery": [
-            "celery>=4.2.2",
+            "celery==4.2.2",
             "redis",
             "importlib-metadata<5.0.0; python_version < '3.8'",  # FIXME: c.f. https://github.com/celery/celery/issues/7783
         ]

